python/hsfs/statistics.py

```diff
@@ -80,13 +125,49 @@ def commit_time(self):
         return self._commit_time
 
     @property
-    def feature_group_commit_id(self):
-        return self._feature_group_commit_id
+    def rowPercentage(self):
```

Comment:
We should copy the setter from monitoring config

Reply:
Done